### DISCUSSION TITLE
Trivy operator different scan result for image then latest trivy scan

### DISCUSSION THREAD

**Discussion by patel-aum** [aquasecurity/trivy-operator#1909] [Help & Support]
i am facing an issue that my trivy-operator in cluster is showing more vulnerabilities then trivy tool 
version for trivy-operator is app: 0.18.5 , Chart: 0.20.6
version for trivy is Version: 0.49.1

both are latest only and still there is a discrepancy in vulnerabilities by 35

**Comment by chen-keinan** [comment]
@patel-aum can you add details on how you are running (command) the cli tool and configuration of `trivy-operator` ?

**Reply by patel-aum** [reply]
trivy image --ignore-unfixed <img name>
trivy operator is configured with helm-Prometheus-thanos-grafana stack so when we restart that container running that image it pulls the new image and show the scan report in grafana which is differing from one we received from normal trivy cli report

**Reply by chen-keinan** [reply]
`trivy-operator` by default do not run with [--ignore-unfixed](https://github.com/aquasecurity/trivy-operator/blob/main/deploy/helm/values.yaml#L382) flag , you'll have to set it, I assume afterwards you'll get the same results

**Reply by patel-aum** [reply]
Yes it's set in operator also

**Reply by chen-keinan** [reply]
can you share trivy-operator configmaps ? and deployment EnvVar ?

**Reply by patel-aum** [reply]
trivy operator trivy-values.yaml:

serviceMonitor:
  enabled: true
trivy:
  ignoreUnfixed: true

**Reply by chen-keinan** [reply]
can you share the actual config files and remove any sensitive information?
